### DISCUSSION TITLE
posts/2022/06/static-tweets-hugo-update/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#20] [Announcements]
# posts/2022/06/static-tweets-hugo-update/

Now!! With new features!! This Hugo shortcode allows you to embed tweets with link “cards.”

https://www.brycewray.com/posts/2022/06/static-tweets-hugo-update/

**Comment by mettamyron** [comment]
Nice! Is it possible to do this in Astro? With the public syndication link?

**Reply by brycewray** [reply]
@maitrimyron I believe [`astro-embed`](https://github.com/astro-community/astro-embed) is what you’re seeking, particularly its [`astro-embed-twitter`](https://github.com/astro-community/astro-embed/tree/main/packages/astro-embed-twitter) package. And, yes, it uses the public syndication link.

(See also my post, “[Static tweets in Astro](https://www.brycewray.com/posts/2022/04/static-tweets-astro/),” which gave the Astro team a few ideas about the initial form of `astro-embed-twitter`. Of course, their result ended up being **far** superior to what I’d done.)

**Reply by mettamyron** [reply]
Thanks again @brycewray! Would love to get your quoted tweet and card working with it.
